Source/DataSources/ModelGraphics.js

```diff
@@ -190,6 +191,14 @@ define([
          */
         runAnimations : createPropertyDescriptor('runAnimations'),
 
+        /**
+         * Gets or sets the numeric Property specifying the relative speed to play the glTF animations.
+         * @memberof ModelGraphics.prototype
+         * @type {Property}
+         * @default 1.0
+         */
+        speedupAnimations : createPropertyDescriptor('speedupAnimations'),
```

Comment:
This should probably be called `animationSpeedMultiplier` or something like that

Reply:
Ah, I see it's already called `speedup` in `ModelAnimation`.  Is that a common term in the animation world?  We might want to rename it otherwise.  `speedup` has no meaning to me, but I know intuitively what a `multiplier` is.

Reply:
Yeah I wasn't happy with `speedup` either, but I guess it would be a breaking change to rename that now. 

The original intent might have been to use this variable to speed up animations but multiplier makes more sense since you can slow it down too.

Reply:
Cool.  Can you open a separate PR deprecating `speedup` and replacing it with `animationMultiplier`?  It's not worth adding new weird attribute names because we had a weird name for an old one.